WordPress/src/main/java/org/wordpress/android/ui/compose/views/TrainOfAvatarsView.kt

```diff
@@ -0,0 +1,140 @@
+import org.wordpress.android.ui.compose.theme.AppTheme
+import org.wordpress.android.util.DisplayUtils
+import org.wordpress.android.util.GravatarUtils
+
+@Suppress("MemberVisibilityCanBePrivate", "unused")
+class TrainOfAvatarsView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0
+) : AbstractComposeView(context, attrs, defStyleAttr) {
+    private val avatarsState: MutableState<List<AvatarItem>> = mutableStateOf(emptyList())
+
+    // icon size in DP used by the Composable
+    private val iconSizeState: MutableState<Int> = mutableStateOf(DEFAULT_ICON_SIZE_DP)
+
+    // icon border width in DP used by the Composable
+    private val iconBorderWidthState: MutableState<Int> = mutableStateOf(DEFAULT_ICON_BORDER_WIDTH_DP)
+
+    // placeholder drawable/color resource to be used by the Composable
+    private val placeHolderResState: MutableState<Int> = mutableStateOf(R.color.placeholder)
+
+    var avatars: List<AvatarItem>
+        get() = avatarsState.value
+        set(value) {
+            if (avatarsState.value != value) avatarsState.value = value
+        }
+
+    /**
+     * Icon size in pixels
+     */
+    var iconSize: Int
+        get() = DisplayUtils.dpToPx(context, iconSizeState.value)
+        set(value) {
+            val iconSizeDp = DisplayUtils.pxToDp(context, value)
+            if (iconSizeState.value != iconSizeDp) iconSizeState.value = iconSizeDp
+        }
+
+    /**
+     * Icon border width in pixels
+     */
+    var iconBorderWidth: Int
+        get() = DisplayUtils.dpToPx(context, iconBorderWidthState.value)
+        set(value) {
+            val borderWidthDp = DisplayUtils.pxToDp(context, value)
+            if (iconBorderWidthState.value != borderWidthDp) iconBorderWidthState.value = borderWidthDp
+        }
+
+    var placeHolderRes: Int
+        get() = placeHolderResState.value
+        set(value) {
+            if (placeHolderResState.value != value) placeHolderResState.value = value
+        }
+
+    init {
+        val ta = context.obtainStyledAttributes(attrs, R.styleable.TrainOfAvatarsView)
+        ta.getDimensionPixelSize(R.styleable.TrainOfAvatarsView_iconSize, -1)
+            .takeIf { it != -1 }
+            ?.let { iconSize = it }
+        ta.getDimensionPixelSize(R.styleable.TrainOfAvatarsView_iconBorderWidth, -1)
+            .takeIf { it != -1 }
+            ?.let { iconBorderWidth = it }
+        ta.getResourceId(R.styleable.TrainOfAvatarsView_placeholder, -1)
+            .takeIf { it != -1 }
+            ?.let { placeHolderRes = it }
+        ta.recycle()
+    }
+
+    @Composable
+    override fun Content() {
+        if (avatarsState.value.isEmpty()) return
+
+        AppTheme {
+            TrainOfIcons(
+                iconModels = avatarModels(),
+                iconSize = iconSizeState.value.dp,
+                iconBorderWidth = iconBorderWidthState.value.dp,
+                placeholder = placeholderPainter()
```

Comment:
This is not a color necessarily, it can be any painter, so `placeholder` makes sense here, which is how it's declared in Coil's `AsyncImage`, but maybe it could be `placeholderPainter` to be clearer, wdyt?